src/errors/test-run/templates.js

```diff
@@ -90,6 +90,10 @@ export default {
         ${err.instantiationCallsiteName} execution was interrupted by page unload. This problem may appear if you trigger page navigation from ${err.instantiationCallsiteName} code.
     `,
 
+    [TEST_RUN_ERRORS.clientFunctionExecutionTimeoutError]: err => `
+        ${err.instantiationCallsiteName} execution was interrupted by timeout.
```

Comment:
The `skipJsError` feature automatically get timeout after the [timeout for ClientFunction](https://github.com/DevExpress/testcafe/issues/3980) option will be implemented. So, we can remove the separate implementation for it right now.